scripts/apps/search/directives/ItemList.js

```diff
@@ -808,9 +811,13 @@ export function ItemList(
             });
 
             var ListItemInfo = function(props) {
+                var itemInfoClassName = 'item-info';
+                if (config.reducedItemListRowHeight) {
+                    itemInfoClassName = 'item-info item-info-reduced-rowheight';
```

Comment:
Here, I would prefer:
```js
let cx = 'item-info' + config.list.narrow ? ' item-info-reduced-rowheight' : '';
```
or
```
if (config.list.narrow) {
    cx += ' item-info-reduced-rowheight';
}
```
`cx` is a common used name in React applications for class names. Or at most `className(s)`, imo `itemInfoClassName` is a bit repetitive.